nntrainer/compiler/onnx_interpreter.h

```diff
@@ -36,14 +40,58 @@ class ONNXInterpreter : public GraphInterpreter {
    * @brief Construct a new ONNXInterpreter object
    *
    */
-  ONNXInterpreter(){};
+  ONNXInterpreter() { registerNodeHandlers(); };
```

Comment:
I think the ONNXInterpreter should have only one registry to handle these NodeHandler. May be you could use singleton.